pkgs/tools/misc/pandoc-include/default.nix

```diff
@@ -10,14 +10,14 @@
 
 buildPythonApplication rec {
   pname = "pandoc-include";
-  version = "1.3.3";
+  version = "1.4.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "DCsunset";
     repo = "pandoc-include";
     rev = "refs/tags/v${version}";
-    hash = "sha256-8eTK7jTwxgSjlxEzjFlL5mzt4tBolmS6fSRnbXDWYJc=";
+    hash = "sha256-rhA+mPjwMoWwX0VIoSZ3aRmlgUy88eGyjSuM3BhyAAQ=";
   };
 
   nativeBuildInputs = [
```

Comment:
```suggestion
  build-system = [
```

Reply:
Should I open another PR for this change? I don't think I have permission to commit to this branch

Reply:
Yes